__init__.py

```
@@ -1,26 +1,36 @@
+#
+
+# FUNCTIONS -> DEBUG
+def debug(*variables: object):
+    for variable in variables:
+        print(f"DEBUG: {repr(variable)}", file=sys.stderr)
+
+# FUNCTIONS -> ADD
+def add(*extensions: str) -> str:
+    code = []
+    for extension in extensions:
+        with open(f"/srv/www/website/extensions/{extension}.py", "r") as file: code.append(file.read())
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)